src/clandinin_lab_to_nwb/brezovec/brezovecnwbconverter.py

```diff
@@ -23,3 +23,12 @@ class BrezovecNWBConverter(NWBConverter):
         ImagingAnatomicalRed=BrezovecAnatomicalRedImagingInterface,
         Video=VideoInterface,
     )
+
+    def temporally_align_data_interfaces(self):
```

Comment:
I think you should do two things here:
1) Of all the datetimes that you have in the interface (or possibly from somewhere else) you choose the correct one and then set it as the `session_session_start_time` of the NWBFile in the metadata.
2) Take all the `start_times` that you have and then substract the time component of the correct `session_start_time` (which is a date) from them so they are aligned with respect to it.